dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ConjureBodySerDe.java

```diff
@@ -78,13 +78,8 @@
     @Override
     public Deserializer<Void> emptyBodyDeserializer() {
         return input -> {
-            try (Response response = input) {
-                if (response.body().read() != -1) {
-                    throw new SafeRuntimeException("Expected empty response body");
-                }
-            } catch (IOException e) {
-                throw new SafeRuntimeException("Failed to read from response body", e);
-            }
+            // We should not fail if a server that previously return nothing starts returning a response
```

Comment:
nit: grammar is a little funky here

Reply:
was there a bug here previously w.r.t. the missing `input.close()`? if so, can you add a test?

Reply:
No there was not a bug, it was previously closed by the try-with-resource. It was simply more concise to call close directly